src/libs/ui/docsetsdialog.cpp

```diff
@@ -572,6 +574,8 @@ void DocsetsDialog::setupAvailableDocsetsTab()
     using Registry::DocsetRegistry;
 
     ui->availableDocsetList->setItemDelegate(new ProgressItemDelegate(this));
+    ProgressItemDelegate* del = static_cast<ProgressItemDelegate*>(ui->availableDocsetList->itemDelegate());
```

Comment:
Just create delegate first, and then set it, then the'd be no need in casting.

Also coding style: `ProgressItemDelegate *delegate`.